app/Jobs/SendSignupToPhoenix.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Rogue\Jobs;
+
+use Rogue\Models\Signup;
+use Rogue\Services\Phoenix;
+use Rogue\Services\Registrar;
+use Illuminate\Queue\SerializesModels;
+use Illuminate\Queue\InteractsWithQueue;
+use Illuminate\Contracts\Queue\ShouldQueue;
+
+class SendSignupToPhoenix extends Job implements ShouldQueue
+{
+    use InteractsWithQueue, SerializesModels;
+
+    /*
+     * Signup instance.
+     *
+     * @var Rogue\Models\Signup
+     */
+    protected $signup;
+
+    /**
+     * Create a new job instance.
+     *
+     * @return void
+     */
+    public function __construct(Signup $signup)
+    {
+        $this->signup = $signup;
+    }
+
+    /**
+     * Execute the job.
+     *
+     * @return void
+     */
+    public function handle(Phoenix $phoenix, Registrar $registrar)
```

Comment:
Dependency injection should happen [in the `handle` method](https://laravel.com/docs/5.3/queues#class-structure) for jobs. The constructor is for parameters that get serialized into the job (the specific signup that's being processed and is passed in when you dispatch `new SendSignupToPhoenix($signup)`)